### PR TITLE
RH-supported SCAP contents

### PR DESCRIPTION
**Pull request by Lennonka**
RHEL hosts have to be scanned with a supported SCAP content, which is the SSG designated for the minor RHEL version.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/common/modules/proc_loading-the-default-scap-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+SSG is provided by the {ProjectServer}'s operating system and installed in `/usr/share/xml/scap/ssg/content/`.
+Note that the available data streams depend on the operating system version on which {Project} runs.
+ifdef::satellite[]
+You can only use this SCAP content to scan hosts that have the same minor RHEL version as your {ProjectServer}.
+For more information, see xref:getting-supported-scap-contents-for-rhel_{context}[].
+endif::[]
+
+.Prerequisites
+* Your user account has a role assigned that has the `create_scap_contents` permission.
```
[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/common/modules/proc_loading-the-default-scap-contents.adoc:14]: I'm keeping it for consistency with other modules.

**Review by adamlazik1 — APPROVED** [review]
LGTM stylewise.

**Comment by ares** [comment]
thumbs up content wise

**Comment by Lennonka** [comment]
Cherry-picked:
- 2c8de6372..cf8a3cb96  3.9 -> 3.9
- 8ac7a35a1..7e66ebe82  3.8 -> 3.8
- ae6a922e7..acfe650e5  3.7 -> 3.7
- 3a8b1b7a4..fa93011ce  3.6 -> 3.6
- e5db2b986..96bfb8985  3.5 -> 3.5
- f09ad1cea..7b901130e  3.4 -> 3.4
- b3bb917eb..5e2010839  3.3 -> 3.3
- 521f69927..5e77e32ec  3.2 -> 3.2
- a8772b25f..9c55679bf  3.1 -> 3.1

**Pull request merged**